core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -323,37 +325,47 @@ __wasi_errno_t
 wasmtime_ssp_clock_res_get(__wasi_clockid_t clock_id,
                            __wasi_timestamp_t *resolution)
 {
-#ifdef BH_PLATFORM_WINDOWS
-    return __WASI_ENOSYS;
+
+    #ifdef BH_PLATFORM_WINDOWS
+    //os_win_clock_res_get(clock_id, resolution);
+
+
+
 #else
-    clockid_t nclock_id;
-    if (!convert_clockid(clock_id, &nclock_id))
-        return __WASI_EINVAL;
-    struct timespec ts;
-    if (clock_getres(nclock_id, &ts) < 0)
-        return convert_errno(errno);
-    *resolution = convert_timespec(&ts);
+    {
```

Comment:
everything in the #else should probably be moved to the `os_clock_res_get` function in the `posix_time.c`

Reply:
Including the implementation of convert_errno and convert_timespec (based on previous comment)